### PR TITLE
Issue #3473 - decrease pollInterval on node policy change

### LINKED ISSUE #3473: Bug: pollInterval is too long when there is a node policy change
### Describe the bug.

A user runs `hzn policy update --input-file=/tmp/changes.json` to change the node policy.
Any new agreement / service container start is delayed for a really long time (70 seconds).

There is a UPDATE_TYPE_ALERT section in `changes/changes_worker.go` which updates the polling interval to (min + max)/ POLL_INTERVAL_ALERT_LEVEL
By default, this will be (20+120)/2 = 70.  Essentially for 70 seconds the edge node does nothing even though it has pending node policy or userinput changes.

Suggested fix would be to modify POLL_INTERVAL_ALERT_LEVEL from `2` to `4`

That will decrease the delay to 35 seconds.

### Describe the steps to reproduce the behavior.

`hzn policy update --input-file=/tmp/changes.json`

Watch the logs. The agent does a PUT to the exchange
```
I0923 19:53:01.206644      22 worker.go:585] MessageDispatcher: Handling Message (*events.NodePolicyMessage): event: UPDATE_POLICY
I0923 19:53:01.206762      22 worker.go:593] MessageDispatcher: Success handling message: propagated event to all workers
I0923 19:53:01.206809      22 worker.go:296] CommandDispatcher: Agreement received command (*agreement.NodePolicyChangedCommand): {event: UPDATE_POLICY}
I0923 19:53:01.206894      22 worker.go:296] CommandDispatcher: Governance received command (*governance.NodePolicyChangedCommand): {event: UPDATE_POLICY}
I0923 19:53:01.206908      22 worker.go:310] CommandDispatcher: Agreement handled command (*agreement.NodePolicyChangedCommand)
I0923 19:53:01.206929      22 worker.go:296] CommandDispatcher: ExchangeChanges received command (*changes.UpdateIntervalCommand): UpdateIntervalCommand: UpdateType: ALERT
I0923 19:53:01.206947      22 changes_worker.go:453] Exchange Changes Worker: Setting poll interval to alert level 70, max interval is 120, increment is 20.
I0923 19:53:01.206957      22 worker.go:310] CommandDispatcher: ExchangeChanges handled command (*changes.UpdateIntervalCommand)
I0923 19:53:01.206966      22 worker.go:338] CommandDispatcher: Agreement command processor blocking for commands
I0923 19:53:01.206976      22 worker.go:348] CommandDispatcher: ExchangeChanges command processor non-blocking for commands
I0923 19:53:01.207894      22 worker.go:310] CommandDispatcher: Governance handled command (*governance.NodePolicyChangedCommand)
```
Nothing meaningful happens in the log for 70 seconds, then:
```
I0923 19:54:11.207456      22 changes_worker.go:255] Exchange Changes Worker: looking for changes starting from ID 21402156
I0923 19:54:11.207536      22 changes.go:248] Exchange RPC getting 1000 changes since change ID 21402156 in orgs []
I0923 19:54:11.207601      22 rpc.go:984] Exchange RPC Invoking exchange POST at https://mgmt-hub-url/edge-exchange/v1/orgs/myorg/changes with {21402156 1000 []}
I0923 19:54:11.220034      22 governance.go:1400] GovernanceWorker: GovernanceWorker dispatching no work handler.
I0923 19:54:11.220088      22 governance.go:400] GovernanceWorker: governing pending agreements
I0923 19:54:11.221058      22 worker.go:348] CommandDispatcher: Governance command processor non-blocking for commands
I0923 19:54:11.464437      22 changes.go:286] Exchange RPC found 2 changes since ID 21402156 with latest change ID 21402179 in orgs []
```

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating Environment

Linux

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by johnwalicki**
Signed-off-by: John Walicki <johnwalicki@gmail.com>

# Pull Request Template

## Description

Change the `POLL_INTERVAL_ALERT_LEVEL` from `2` to `4` 
This will decrease the pollInterval when there is a pending node policy change from 70 seconds to 35 seconds

I also fixed a variety of minor spelling issues in the comments and strings.

Fixes #3473

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Run `hzn policy update --input-file=/tmp/changes.json` and watch the logs.
...

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: changes/changes_worker.go [hunk 1]
```diff
@@ -430,27 +430,26 @@ func (w *ChangesWorker) handleHeartbeatStateAndError(changes *exchange.ExchangeC
...
 			w.SetNoWorkInterval(w.pollInterval)
 			glog.V(3).Infof(chglog(fmt.Sprintf("Resetting poll interval to %v, max interval is %v, increment is %v.", w.pollInterval, w.pollMaxInterval, w.pollAdjustment)))
 		}
-
 		w.noMsgCount = 0
 	} else if updateType == UPDATE_TYPE_ALERT {
 		// This is the case when the node should be watching the changes more often than max but not as frequent as min.
 		// It is called alert.
-		// For excample: node policy changed, node userinput changed, node itself changed, service changed,
+		// For example: node policy changed, node userinput changed, node itself changed, service changed,
```
Comment (MaxMcAdam) [inline changes/changes_worker.go:452]: Please update the comments here and for UPDATE_TYPE_RESET to reflect this change

**Review by MaxMcAdam — CHANGES_REQUESTED** [review]

**Comment by SanjeevKGupta** [comment]
The possible ramification of unintended consequences by decreasing the polling interval should be looked at thoroughly, For one it will increase the hit to mgmt hub.

File: changes/changes_worker.go (hunk 1 above)
Comment (johnwalicki) [reply changes/changes_worker.go]: Good suggestion - I updated the comments.

**Comment by johnwalicki** [comment]
> The possible ramification of unintended consequences by decreasing the polling interval should be looked at thoroughly, For one it will increase the hit to mgmt hub.

The polling interval will temporarily reset to 20 seconds so that the agreement negotiation will start. It happens on one edge node, most likely when the user runs `hzn policy update`.  The polling interval will back off normally when there is no more work to do.  I don't think it will impact the mgmt hub.

**Comment by SanjeevKGupta** [comment]
Can this be confirmed that this change happens only on one node and is temporary in nature?

**Review by MaxMcAdam — APPROVED** [review]

**Pull request merged**